src/lib/protocols/presentproof/messages/ProofRequest.ts

```diff
@@ -0,0 +1,44 @@
+import { Expose } from "class-transformer/decorators";
+
+export class ProofRequest {
+
+    public constructor(proofRequest: ProofRequestInterface) {
+        if (proofRequest) {
+            this.name = proofRequest.name;
+            this.version = proofRequest.version;
+            this.nonce = proofRequest.nonce;
+            this.requestedAttributes = proofRequest.requestedAttributes;
+            this.requestedPredicates = proofRequest.requestedPredicates;
+        }
+    }
+
+    @Expose({ name: 'name' })
+    public name!: string;
+    @Expose({ name: 'version' })
+    public version!: string;
+    @Expose({ name: 'nonce' })
+    public nonce!: string;
```

Comment:
Same here. `@Expose` is only needed if the json property name is different from the class property name